Makefile

```diff
@@ -72,10 +75,12 @@ commit:
 	git push)
 
 gridmatches: gxd.sqlite src/gridcmp.so
-	cat src/findmatches.sql | time sqlite3 gxd.sqlite
+	time python3 src/findmatches.py -N 100
```

Comment:
This PR is missing src/findmatches.py

Reply:
Nice catch! I thought I committed in the previous PR, but I guess that explains why this one is so small.